apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerOptions.cs

```diff
@@ -157,11 +157,12 @@ public ResourcesExhaustedBehavior ResourcesExhaustedBehavior
         }
 
         /// <summary>
-        /// Defines the global timeout duration.
+        /// Defines the default values for <see cref="SpannerCommand.CommandTimeout"/> and
```

Comment:
Maybe add the part about this being seconds (and 0 having a variable meaning) here so that users don't need to refer to other docs?